griptape/drivers/image_generation_model/stable_diffusion_3_controlnet_pipeline_image_generation_model_driver.py

```diff
@@ -0,0 +1,85 @@
+from __future__ import annotations
+
+import os
+from typing import TYPE_CHECKING, Any, Optional
+
+from attrs import define, field
+
+from griptape.drivers import StableDiffusion3PipelineImageGenerationModelDriver
+from griptape.utils import import_optional_dependency
+
+if TYPE_CHECKING:
+    from PIL.Image import Image
+else:
+    StableDiffusion3ControlNetPipeline = import_optional_dependency(
+        "diffusers.pipelines.controlnet_sd3.pipeline_stable_diffusion_3_controlnet"
+    ).StableDiffusion3ControlNetPipeline
```

Comment:
Can't have this import here as it'll break bare installs.

Reply:
Whoops, leftover test change. Removed.